utils/ssh/v2_ssh.go

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ssh
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/alibaba/sealer/common"
+	"github.com/alibaba/sealer/logger"
+	v2 "github.com/alibaba/sealer/types/api/v2"
+	"github.com/alibaba/sealer/utils"
+	"github.com/imdario/mergo"
+)
+
+type V2Interface interface {
```

Comment:
Should we need change the ssh interface?  I think we just need add a New ssh function: https://github.com/alibaba/sealer/pull/891/files

Reply:
Then new a ssh client by IP and v2.Cluster: https://github.com/alibaba/sealer/pull/882/files#diff-22657dc896bfef614095b0434316b07105c9d66099cb05b60ccbfed54c3fcb8cR94